src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx

```diff
@@ -249,6 +249,7 @@ const UnitsTab: FC<UnitsTabProps> = ({ trackingData, formattedData }) => {
                     aria-label={"None highlighted"}
                     value={""}
                     data-testid={"no-threads-radio"}
+                    isNoneHighlighted={true}
```

Comment:
By adding `isNoneHighlighted` we've made the `<RadioButton/>`'s (SimpleRadioButton) API specific to this component.

Instead we can remove this and add padding to the containing `<Box/>` which I believe is `$pl`